nipype/__init__.py

```diff
@@ -56,3 +58,17 @@ def get_info():
 from .pipeline import Node, MapNode, JoinNode, Workflow
 from .interfaces import (DataGrabber, DataSink, SelectFiles, IdentityInterface,
                          Rename, Function, Select, Merge)
+
+
+if config.getboolean('execution', 'check_version'):
+    import etelemetry
+
+    latest = {"version": 'Unknown'}
+    try:
+        latest = etelemetry.get_project("nipy/nipype")
```

Comment:
@mgxd @satra It looks like etelemetry lets us send arbitrary kwargs. Should we add version information, so we can get a sense of which versions are still in use? e.g.,

```Python
latest = etelemetry.get_project("nipy/nipype", version=__version__)
```

Reply:
that was intended to allow altering any `requests` specific arguments (such as timeout), but we could leverage `params` to do this.